### PR TITLE
Fixing more ruin bugs! (Done for real now no bamboozles)

### LINKED ISSUE #4481: Lavaland Gas station issues and suggestions
The air tank in the back is rotated in such a way that it does not feed into the air tank in the store.

image

Could air vents that prevent air flow be added to each door so the place doesn't constantly loose pressure/air every time a door is opened? 

I do not know if it was there at the start and just taken later (joined late) but if not, could a backpack be added to the back of the store or have the clerk start with one? Similarly I did not notice any keys for the ATV.

The light switch can be accessed from inside and outside the store.

Ditch the cleanbot and just give a mop and bucket since there is already a sink and mop bucket.

### LINKED ISSUE #4487: Cosmic ashwalker has digurtutate(fuckhowdoispellit) legs
Cosmic starts with a pair of no-slips. He can't wear them because his legs are scuffed. Fluffy forgot to give him proper legs. Blame Fluffy.

### LINKED ISSUE #4526: Gas station clerk and vendors
If the gas station clerk has to leave their place for any reason, then on the station, the vendor menus don't even pop up when they have their ID on.

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑
tweak: Cosmic Ashwalkers can now wear shoes!
bugfix: Chemistry Ruin is no longer filled with ash
tweak: More stuff has been added and or fixed with the gas station ruin!
tweak: Moved The Academy Die of Fate.. Again!!
/🆑

for the Gas Station Ruin they now have a slightly better toolbox, a instrument to play music and stuff, Display Cases to store more valuable things, Emergency Supplies so they can repair the gas station incase someone starts breaking it open and other bugs I fixed from #4481 

So yeah fixes #4481 fixes #4526 and fixes #4487

### CODE REVIEW

**Comment by Time-Green** [comment]
Why do they need to wear shoes? Ashwalkers can’t and cosmic ashwalker is simply an abducted ashwalker.

Unless its feet abductors in wich case it makes sense

**Comment by fluffe9911** [comment]
Because they got a pair of no slips in there locker and since theres no thing in place to make shoes digigrade compatible easily decided to just make it a lizard subtype just think of it as the abductors giving it normal feet since having feet that could not wear shoes would be a disadvantage for what the abductors planned on using the ashwalker for

edit (also just realized this would make them able to use guns as well heck)

edit edit dealt with

**Comment by Time-Green** [comment]
also just remove the digitgrade on cosmic ashwalkers
species_traits = list(MUTCOLORS,EYECOLOR,LIPS)
instead of species_traits = list(MUTCOLORS,EYECOLOR,LIPS,DIGITIGRADE)

**Comment by fluffe9911** [comment]
That is much better solution I did not think of will do

**Review by Altoids1 — APPROVED** [review]
Does not include any pulse rifle pistols, 👍 from me

File: yogstation/code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -66,10 +66,12 @@
...
+	flavour_text = "<span class='big bold'>You are a Gas Station Clerk.</span><b> The gas station you worked most of your life in was moved to some hellhole in the middle of nowhere for some reason, you are to try to make the best of the situation and make as much money as possible from any locals or passerbys you may encounter. Feel free to explore around the planet and find things to sell to potential customers but do not leave the planet unless the gas station is somehow completely destroyed, If someone is trying to break in or is trying to steal your products you have the right to use any means necessary to stop them including murder."
```
Comment (Altoids1) [inline yogstation/code/modules/ruins/lavaland_ruin_code.dm]: You have a \<b\> in there with no ending \</b\>.

**Review by Altoids1 — CHANGES_REQUESTED** [review]

**Review by Altoids1 — APPROVED** [review]

**Pull request merged**